### PR TITLE
Started project and package detail pages

### PR DESCRIPTION
**Pull request by BenCrafterRED**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,14 @@ License
 -------
 
 the LightBuildServer is published with the GNU Lesser General Public License v2 (or 2.1) or later.
+
+Setup for Development
+---------------------
+
+1. Create a Python 3 virtual environment: `LightBuildServer$ python -m venv .venv`
+
+2. Actvate the virtual environment: [See venv documentation](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments)
```
Comment (tpokorra) [inline README.md:45]: i is missing: Activate

File: projects/views.py [hunk 2]
```diff
@@ -13,3 +14,28 @@ def get_queryset(self):
             chain([self.request.user], User.objects.exclude(pk__exact=self.request.user.pk))
                 if self.request.user.is_authenticated else User.objects.all()
         ) ]
+
+class ProjectView(generic.DeleteView):
```
Comment (tpokorra) [inline projects/views.py:18]: is DeleteView correct?

**Review by tpokorra — CHANGES_REQUESTED** [review]
thank you for this pull request!

File: README.md (hunk 1 above)
Comment (BenCrafterRED) [reply README.md:45]: Thank you.

File: projects/views.py (hunk 2 above)
Comment (BenCrafterRED) [reply projects/views.py:18]: No, it should have been DetailView.

**Comment by BenCrafterRED** [comment]
Files have been corrected.

**Review by tpokorra — APPROVED** [review]
great! that was quick :)

**Pull request merged**